js/app.jsx

```diff
@@ -486,6 +486,26 @@ var Job = React.createClass({
       </ul>
     );
 
+    function slimTrace(str) {
+      var lines = [];
+      if (str) {
+        lines = str.split(/,|ÿ/)
+                   .map(val => { return val.trim(); })
+                   .map(val => {
+                    var matches = val.match(/\w*\.\w+\(\w+\.scala:\d+\)/i);
+                    return matches ? matches[0] : val;
+                   });
+      }
+      return (
+        <ul className="list-unstyled">
+          {lines.map(l => <li>{l}</li>)}
```

Comment:
Looking at an example job (job_1467750617173_112937) the same line number is listed twice. We should dedupe.